src/AutoRest.CSharp/LowLevel/AutoRest/LowLevelOutputLibrary.cs

```diff
@@ -24,10 +24,37 @@ public LowLevelOutputLibrary(CodeModel codeModel, BuildContext<LowLevelOutputLib
         {
             _codeModel = codeModel;
             _context = context;
+            UpdateListMethodNames();
             _internalRestClients = new CachedDictionary<OperationGroup, LowLevelRestClient>(EnsureRestClients);
             _publicClients = new CachedDictionary<OperationGroup, LowLevelDataPlaneClient>(EnsureClients);
         }
 
+        private void UpdateListMethodNames()
+        {
+            foreach (var operationGroup in _codeModel.OperationGroups)
+            {
+                foreach (var operation in operationGroup.Operations)
+                {
+                    var curName = operation.Language.Default.Name;
+                    if (curName.Equals("List") || curName.Equals("ListAll"))
+                    {
+                        operation.Language.Default.Name = "GetAll";
```

Comment:
We don't use `GetAll` in .NET.  Please see the .NET Guidelines: https://azure.github.io/azure-sdk/dotnet_introduction.html#general-client-standardize-verbs